.github/workflows/publish-package.yml

```diff
@@ -16,17 +16,19 @@ jobs:
       - run: |
           yarn sdk:lint
           yarn sdk:build
-      - run: cd packages/enums && npm publish
-      - run: cd packages/types && npm publish
-      - run: cd packages/utils && npm publish
-      - run: cd packages/event && npm publish
-      - run: cd packages/bucketing && npm publish
-      - run: cd packages/logger && npm publish
-      - run: cd packages/rules && npm publish
-      - run: cd packages/segments && npm publish
-      - run: cd packages/api && npm publish
-      - run: cd packages/data && npm publish
-      - run: cd packages/experience && npm publish
-      - run: cd packages/js-sdk && npm publish
+      - run: |
+          START=$PWD
+          cd packages/enums && npm publish && cd $START
```

Comment:
this will work @abbaseya  but would be prettier like 
```
- run: (cd packages/enums && npm publish)
- run: (cd packages/types && npm publish)
- ............ 
```
That wont really change current working dir